docs/guide/limitation_reference.md

```diff
@@ -402,6 +402,25 @@ is evaluated.
 For more information on how to restrict User's access to part of the Subtree 
 follow [the example in the Admin management section](admin_panel.md#restrict-editing-to-part-of-the-tree).
 
+## Version Lock Limitation
+
+A Limitation to specify whether the User can perform edit or unlock actions on 
+Content items that are in a specific workflow stage.
```

Comment:
Content item doesn't have to be in any specific state (StageLimitation is responsible for that)